### PR TITLE
US140704 Add sync draft properties

### PR DESCRIPTION
**Pull request by mostlyfabulous**
[US140704](https://rally1.rallydev.com/#/?detail=/userstory/641659231455&fdp=true): Sync visibility between new topic + new forum

### CODE REVIEW

File: src/activities/discussions/DiscussionTopicEntity.js [hunk 1]
```diff
@@ -42,25 +42,6 @@ export class DiscussionTopicEntity extends Entity {
 		return { action, fields };
 	}
 
-	/**
-	 * @summary Checks if topic entity has changed, primarily used for dirty check
-	 * @param {object} topic the topic that's being modified
-	 */
-	equals(topic) {
```
Comment (mostlyfabulous) [inline src/activities/discussions/DiscussionTopicEntity.js:49]: Moved down

File: src/hypermedia-constants.js [hunk 2]
```diff
@@ -677,6 +677,13 @@ export const Actions = {
 		skipSubscribeToParentNotifications: 'skip-subscribe-to-parent-notifications',
 		unsubscribeToParentNotifications: 'unsubscribe-to-parent-notifications'
 	},
+	discussions: {
```
Comment (mostlyfabulous) [inline src/hypermedia-constants.js:680]: Now in alphabetical order

**Review by mdulat — APPROVED** [review]

File: src/hypermedia-constants.js (hunk 2 above)
Comment (dustxd) [reply src/hypermedia-constants.js:684]: I don't think there's really an alphabetical order in this file... 😅

File: src/activities/discussions/DiscussionTopicEntity.js [hunk 3]
```diff
@@ -142,21 +123,67 @@ export class DiscussionTopicEntity extends Entity {
 		];
 		return { action, fields };
 	}
+	/**
+	 * @returns {bool} Sync draft status with topic's parent forum against the draft endpoint
+	 */
+	canSyncDraftWithForum() {
+		return this._entity && this._entity.hasActionByName(Actions.discussions.topic.syncDraftWithForum);
+	}
+	/**
```
Comment (dustxd) [inline src/activities/discussions/DiscussionTopicEntity.js:132]: Not like a big deal but can you add some spacing in between the methods please 👾

**Pull request merged**